Makefile

```diff
@@ -209,9 +211,10 @@ vendor-in-container:
 
 .PHONY: vendor
 vendor:
-	GO111MODULE=on $(GO) mod tidy
-	GO111MODULE=on $(GO) mod vendor
-	GO111MODULE=on $(GO) mod verify
+	$(GO) mod tidy
+	$(GO) mod vendor
+	$(GO) mod verify
+	if test -n "$(strip $(shell go env GOTOOLCHAIN))"; then go mod edit -toolchain none ; fi
```

Comment:
What is this `go env` condition checking for? It’s not _absolutely_ obvious to me, my best guess is “is this a new enough Go to understand GOTOOLCHAIN?”

And if that’s the question, the `go 1.21` directive in `go.mod` ensures the answer is “yes” (or, alternatively, if building with a much older Go which does not enforce the `go 1.21` directive, the build is going to fail on a missing library feature anyway).